eth/beacon/types/casper_slashings.py

```diff
@@ -0,0 +1,62 @@
+import rlp
+from rlp.sedes import (
+    CountableList,
+)
+from typing import (
+    Sequence,
+)
+
+from eth.rlp.sedes import (
+    uint24,
+    uint256,
+)
+
+from .attestation_data import AttestationData
+
+
+class SlashableVoteData(rlp.Serializable):
```

Comment:
I'd suggest we make a new file for `SlashableVoteData`, it will be easier to look up the data structure.